ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -634,94 +655,96 @@ struct FrameworkBuilder {
     }
   }
 
-  private func makeSwiftModuleMap(thinArchives: [Architecture: URL], destination: URL) -> Bool {
+  // TODO: Verify if the array is needed, or we can pass in an individual framework for searching.
+  // Why do we need to parse each one? The argument passed in previously was previously the thin
+  // binary slices, not a single URL.
+  private func makeSwiftModuleMap(slicedFramework: URL, destination: URL) -> Bool {
```

Comment:
This is something that really could use a unit test ... but not right now

Reply:
Agreed 😄